mm2src/coins/utxo/slp.rs

```diff
@@ -1083,16 +1083,31 @@ impl MarketCoinOps for SlpToken {
 
     /// Receives raw transaction bytes in hexadecimal format as input and returns tx hash in hexadecimal format
     fn send_raw_tx(&self, tx: &str) -> Box<dyn Future<Item = String, Error = String> + Send> {
-        let coin = self.clone();
+        let _self = self.clone();
```

Comment:
I like this naming, but it may lead to unused variables. We also name such things like `selfi` :)

Reply:
Yeah that's true, I shouldn't applied `unused variable` standard here. I usually prefer using `this`. Don't know why I made it `_self` here.